packages/frontend-2/components/promo-banners/Wrapper.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div>
+    <PromoBannersBanner
+      v-if="activeBanner"
+      :id="activeBanner.id"
+      :primary-text="activeBanner.primaryText"
+      :secondary-text="activeBanner.secondaryText"
+      :url="activeBanner.url"
+      :image-src="activeBanner.imageSrc"
+      @banner-dismissed="activeBanner && handleDismissed(activeBanner.id)"
+    ></PromoBannersBanner>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useSynchronizedCookie } from '~/lib/common/composables/reactiveCookie'
+import dayjs from 'dayjs'
+import type { PromoBanner } from '~/lib/promo-banners/types'
+
+const props = defineProps<{
+  banners: PromoBanner[]
+}>()
+
+const hideAllPromoBanners = useSynchronizedCookie('hide-all-promo-banners', {
+  default: () => false,
```

Comment:
You don't necessarily always have to set defaults for cookies - set them when you actually want them to have a value. In this case it's enough for the cookie to have an undefined value, and only it being created when you set it to `true`